x-pack/plugins/enterprise_search/public/applications/enterprise_search_content/components/new_index/method_connector/new_connector_template.tsx

```diff
@@ -54,44 +54,17 @@ export const NewConnectorTemplate: React.FC<Props> = ({
   type,
   isBeta,
 }) => {
-  const { fullIndexName, fullIndexNameExists, fullIndexNameIsValid, rawName } =
-    useValues(NewConnectorLogic);
+  const { connectorName, rawName } = useValues(NewConnectorLogic);
   const { setRawName } = useActions(NewConnectorLogic);
 
   const handleNameChange = (e: ChangeEvent<HTMLInputElement>) => {
     setRawName(e.target.value);
     if (onNameChange) {
-      onNameChange(fullIndexName);
+      onNameChange(connectorName);
     }
   };
 
-  const formInvalid = !!error || fullIndexNameExists || !fullIndexNameIsValid;
-
-  const formError = () => {
```

Comment:
We're removing this error because we're handling these cases on the backend now by generating a name instead of erroring?

Reply:
no. These are errors from index creation. This page is used for connector creation which shouldn't create an index and it is quite permissive. 

Actually I am not sure if this page is used at all after this PR. I may create another one to remove unused ones later.